storagezone_add.go

```diff
@@ -0,0 +1,32 @@
+package bunny
+
+import "context"
+
+// StorageZoneAddOptions are the request parameters for the Get Storage Zone API endpoint.
+//
+// Bunny.net API docs: https://docs.bunny.net/reference/storagezonepublic_add
+type StorageZoneAddOptions struct {
+	// The origin URL of the storage zone where the files are fetched from.
+	OriginURL *string `json:"OriginUrl,omitempty"`
+	// The ID of the storage zone that the storage zone is linked to.
+	Region *string `json:"Region,omitempty"`
+
+	// The name of the storage zone (Optional)
+	Name *string `json:"Name,omitempty"`
+	// The code of the main storage zone region (Optional)
+	ReplicationRegions []string `json:"ReplicationRegions,omitempty"`
+}
```

Comment:
I would not define an enum (aka in Golang an own type + consts with that type) in this case because it's possible that the enum will be incomplete. This would make it a bit weird to use.
I would only define strings consts like:
```go
const (
  StorageZoneRegionDE = "DE"
  StorageZoneRegionNY = "NY"
...
)
const (
  StorageZoneReplicationRegionDE = "DE"
...
)
```